### PR TITLE
Release Notes for WebView2 120 Prerelease SDK

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article section for review:
https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes?branch=pr-en-us-2870#102164-prerelease

AB#47053647

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -164,6 +164,53 @@ No additional APIs have been promoted to Stable and added in this Release SDK.
...
+
+[NuGet package for WebView2 SDK 1.0.2164-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2164-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 120.0.2164.0 or higher.
+
+
+<!-- ------------------------------ -->
+#### Experimental APIs
+
+* Added the `FailureSourceModulePath` property to the `ProcessFailedEventArgs` type, to specify the full path of the module that caused the crash in cases of Windows code integrity failures - that is, when a process exited with `STATUS_INVALID_IMAGE_HASH`.  See also [Enable code integrity](/mem/intune/user-help/you-need-to-enable-code-integrity) in the Microsoft Intune docs.
```
Comment (victorhuangwq) [inline microsoft-edge/webview2/release-notes.md:180]: I added this link initially, but I think on second thought, let's remove the last line `See also [Enable code integrity](/mem/intune/user-help/you-need-to-enable-code-integrity) in the Microsoft Intune docs.` from this. The link does not provide very helpful information, and having Intune appear here might be confusing.

**Review by victorhuangwq — CHANGES_REQUESTED** [review]
One small change requested. Otherwise looks good

File: microsoft-edge/webview2/release-notes.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md:180]: removed:
`See also [Enable code integrity](/mem/intune/user-help/you-need-to-enable-code-integrity) in the Microsoft Intune docs.`

**Review by peiche-jessica — APPROVED** [review]

**Review by victorhuangwq — APPROVED** [review]
LGTM

**Pull request merged**